crates/grim-channel/README.md

```
@@ -34,18 +35,49 @@ Player-facing verbs and where to find their handlers.
 ## Resources & Events
 | Name | Kind (Resource/Message) | File |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)